tcms/templates/include/datetimepicker_script.html

```diff
@@ -2,7 +2,7 @@
 $(function () {
     $('{{ selector }}').datetimepicker({
         locale: '{{ LANGUAGE_CODE }}',
-        format: 'L',
+        format: 'YYYY-MM-DD',
```

Comment:
Can you explain why we need this change b/c this include file is used in multiple places.

Reply:
Oh, because without this the field does not correspond to the expected format of the DateTime that we are using it is DD/MM/YYYY. I have checked in a few places and it seems okay, but not sure. Do you think this should not be touched? 
![Screen Shot 2021-03-28 at 9 24 20 PM](https://user-images.githubusercontent.com/27647952/112765121-fbe41180-900b-11eb-8d44-e19fc632e422.png)